[2022-05-22 03:27:26,538] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Teste_ETL_Raízen.download_file_xls manual__2022-05-22T02:35:48.168632+00:00 [queued]>
[2022-05-22 03:27:26,650] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Teste_ETL_Raízen.download_file_xls manual__2022-05-22T02:35:48.168632+00:00 [queued]>
[2022-05-22 03:27:26,650] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-05-22 03:27:26,650] {taskinstance.py:1357} INFO - Starting attempt 6 of 6
[2022-05-22 03:27:26,650] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-05-22 03:27:26,670] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): download_file_xls> on 2022-05-22 02:35:48.168632+00:00
[2022-05-22 03:27:26,676] {standard_task_runner.py:52} INFO - Started process 111 to run task
[2022-05-22 03:27:26,679] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Teste_ETL_Raízen', 'download_file_xls', 'manual__2022-05-22T02:35:48.168632+00:00', '--job-id', '64', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Raizen.py', '--cfg-path', '/tmp/tmphwinrkeu', '--error-file', '/tmp/tmpwtsmnkmv']
[2022-05-22 03:27:26,680] {standard_task_runner.py:80} INFO - Job 64: Subtask download_file_xls
[2022-05-22 03:27:26,724] {task_command.py:369} INFO - Running <TaskInstance: Teste_ETL_Raízen.download_file_xls manual__2022-05-22T02:35:48.168632+00:00 [running]> on host 541443bb6989
[2022-05-22 03:27:26,786] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Teste_ETL_Raízen
AIRFLOW_CTX_TASK_ID=download_file_xls
AIRFLOW_CTX_EXECUTION_DATE=2022-05-22T02:35:48.168632+00:00
AIRFLOW_CTX_TRY_NUMBER=6
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-22T02:35:48.168632+00:00
[2022-05-22 03:27:28,647] {python.py:173} INFO - Done. Returned value was: None
[2022-05-22 03:27:28,685] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Teste_ETL_Raízen, task_id=download_file_xls, execution_date=20220522T023548, start_date=20220522T032726, end_date=20220522T032728
[2022-05-22 03:27:28,740] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-05-22 03:27:28,785] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
